typeset/env

```diff
@@ -0,0 +1 @@
+LOG_LEVEL=info
```

Comment:
Can this file be named `.env.example`? That way it is a little clearer that this is not actually being used and other packages like https://www.npmjs.com/package/dotenv-safe look for  that file to validate that the `.env` file is valid